drivers/dma/dma_stm32.c

```diff
@@ -93,24 +93,45 @@ static void dma_stm32_irq_handler(void *arg)
 	}
 
 	stream = &data->streams[id];
+#ifndef CONFIG_DMAMUX_STM32
```

Comment:
would be nicer if you used if (IS_ENABLED()) in relevant places  (not possible for mux_channel attribute though)

Reply:
changed in dma_stm32_reload and in dma_stm32_reload functions